Makefile

```diff
@@ -51,13 +53,29 @@ test-coverage:
 	poetry run coverage report --show-missing
 	poetry run coverage xml
 
+generate-man:
+	@export QPC_VAR_CURRENT_YEAR=$(shell date +'%Y') \
+	&& export QPC_VAR_PROJECT=$${QPC_VAR_PROJECT:-Quipucords} \
+	&& export QPC_VAR_PKG_SOURCE=$${QPC_VAR_PKG_SOURCE:-qpc} \
+	&& jinja -X QPC_VAR docs/source/man.j2 $(ARGS)
+
+update-man.rst:
+	$(MAKE) generate-man ARGS="-o docs/source/man.rst"
+
+manpage-test:
+	@poetry run $(MAKE) generate-man | grep -v "Entering directory" | \
+	grep -v "Leaving directory" > output.txt
+	@diff -u docs/source/man.rst output.txt
+
 manpage:
-	$(pandoc) docs/source/man.rst \
-	  --standalone -t man -o docs/qpc.1 \
-	  --variable=section:1 \
-	  --variable=date:'$(BUILD_DATE)' \
-	  --variable=footer:'version $(PKG_VERSION)' \
-	  --variable=header:'QPC Command Line Guide'
+	@$(MAKE) generate-man | grep -v "Entering directory" | \
+	grep -v "Leaving directory" | \
+	$(pandoc) -s - \
+	  --standalone -t man -o docs/$(QPC_VAR_PKG_SOURCE).1\
+	  --variable=section:1\
+	  --variable=date:'$(BUILD_DATE)'\
+	  --variable=footer:'version $(PKG_VERSION)'\
+	  --variable=header:'$(QPC_VAR_PKG_SOURCE_UPPER) Command Line Guide'
```

Comment:
Should the name really be forced to upper-case in strings like this?

Asking because I do not know the brand standards here, but changing the case of a program name based on documentation context feels strange to me. Maybe this is a question for @bluemorpho-mpw?

Reply:
See also a needlessly pedantic discussion in Slack here: https://redhat-internal.slack.com/archives/C02QSNF1UKE/p1687550700970519

I'm sorry I asked! 😂 

Feel free to resolve this conversation and keep what we have if no strong recommendations come from that thread.

Reply:
I dont think we reached a conclusion there, I will resolve this.